orderandprops.cc

```diff
@@ -252,23 +252,23 @@ void OrderAndProps::describeDictionary( DictListWidget * lst, QModelIndex const
 void OrderAndProps::contextMenuRequested( const QPoint & pos )
 {
   QMenu menu( this );
-  QAction * sortNameAction = new QAction( tr( "Sort by name" ), &menu );
-  menu.addAction( sortNameAction );
-  QAction * sortLangAction = new QAction( tr( "Sort by languages" ), &menu );
-  menu.addAction( sortLangAction );
-
-  QAction * showHeadwordsAction = NULL;
-
   QModelIndex idx = ui.searchLine->mapToSource( ui.dictionaryOrder->indexAt( pos ) );
   sptr< Dictionary::Class > dict;
   if( idx.isValid() && (unsigned)idx.row() < ui.dictionaryOrder->getCurrentDictionaries().size() )
     dict = ui.dictionaryOrder->getCurrentDictionaries()[ idx.row() ];
+
+  QAction * showHeadwordsAction = NULL;
```

Comment:
Why move this line down? It doesn't seem to be related to the actual fix, and it is not an objective improvement, I think.